pages/profile/[address].tsx

```diff
@@ -20,8 +32,25 @@ const Profile: React.FC<Props> = ({ address: addressURL }) => {
   const { address, chainId } = useWallet();
   const isLoggedInUser = addressURL === address?.toLowerCase();
   const { ens } = useENS(address);
+  const [registered, setRegistered] = useState<boolean>(false);
   const profile = formatAddress(address, ens);
 
+  const getPOH = async () => {
+    const client = createClient({
+      url: PoHAPI,
+    });
+    const data = await client
+      .query(getRegisteredStatus, { id: addressURL })
+      .toPromise();
+
+    setRegistered(data?.data?.submission?.registered || false);
+  };
+
+  useEffect(() => {
+    getPOH();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
we should either put the lines inside `getPOH` method into this `useEffect` or wrap the `getPOH` method in a useCallback

Reply:
We could also create a simple hook that takes the `address` and returns a `registered` boolean.